index.d.ts

```diff
@@ -129,6 +129,12 @@ export interface ReactJsonViewProps {
    * Default: false
    */
   sortKeys?: boolean;
+  /**
+   * Set to a value to be used as defaultValue when adding new key to json
+   *
+   * Default: null
+   */
+  defaultValue?: string | number | boolean | null | undefined | string[] | number[] | boolean[] | {}[] | {};
```

Comment:
remove `undefined`. `defaultValue?` means optional already.
I suggest you create a new type like 
```ts
type TypeDefaultValue = string | number | boolean | object;
```
and then, you can define the type for `defaultValue` like
```ts
defaultValue?: TypeDefaultValue | TypeDefaultValue[] | null;
```

Please note that the use of `{}` is discouraged. See the following example:

```ts
type t = {};
const c:t = 1; // no error
```
while
```ts
type t = object;
const c:t = 1; // error
```